fedora_messaging/cli.py

```diff
@@ -452,3 +458,28 @@ def record(exchange, queue_name, routing_key, app_name, limit, file):
     _consume(
         exchange, queue_name, routing_key, messages_recorder.collect_message, app_name
     )
+
+
+URL_TEMPLATE = "https://apps.fedoraproject.org/datagrepper/id?id={}&is_raw=true"
+
+
+@cli.command()
+@click.argument("message_id")
+@click.option("--datagrepper_url", help=_datagrepper_help, default=URL_TEMPLATE)
+def replay(message_id, datagrepper_url):
+    """Replay a message from Datagrepper by its message ID"""
+    try:
+        message_data = _get_message(message_id, datagrepper_url)
+        if message_data:
+            api.publish(message.load_message(message_data))
+            click.echo(f"Message with ID {message_id} has been successfully replayed.")
+    except requests.HTTPError as e:
```

Comment:
Actually this exception can only happen during the `_get_message()` call, so it should not contain the rest (the `if` block).